doc/releases/changelog-dev.md

```diff
@@ -316,6 +316,9 @@
 
 <h4>Other improvements</h4>
 
+* The `QSVT` operation now has a `data` property.
```

Comment:
```suggestion
* The `QSVT` operation now determines it's `data` from the block encoding and projector operator data.
```

Technically it had a `data` property. It just wasn't really what it should have been.